_data/hipaa-landing-page-tabs-code.yml

```diff
@@ -27,9 +31,12 @@
 - id: database
   title: database.js
   content: |
-    <figure><figcaption>Use database secrets to connect securely</figcaption>
+    <figure>
       <pre>
       <code class="language-javascript">
+        /****************************************************
+        * How to use config to connect to a database securely
+        *****************************************************/
```

Comment:
Nit: should there be a newline after these comment blocks just like the first above?

Reply:
Good catch! Added in 67ed3b7